File: card-classes/brief/render.py
import mica
import markdown

@mica.render
def render(content, config):
    """Render the agent brief card."""
    html = markdown.markdown(content, extensions=["tables", "fenced_code", "sane_lists"])
    return f'''
    <div class="card-brief">
        <div class="card-markdown">{html}</div>
    </div>
    '''

File: card-classes/brief/render.js
/**
 * Brief card class — renders the agent brief.
 */

import { marked } from 'marked';

export default function render(content, config) {
  const html = marked.parse(content, { breaks: false, gfm: true });
  return `
    <div class="card-brief">
      <div class="card-markdown">${html}</div>
    </div>
  `;
}
